hyperbahn/clients/logger.js

```diff
@@ -21,9 +21,13 @@
 'use strict';
 
 var os = require('os');
-var Logger = require('logtron');
+var Logtron = require('logtron');
```

Comment:
I think it would be good to retain the principle that brand names from npm only infect the first component of the top-level module identifier. I would keep this line as-is. Keep larch in the module identifier in anticipation of breaking it into an npm package, but: `var ReservoirLogBackend = require('../lib/larch/reservoir-backend');` etc. This will make the purpose more obvious to the casual observer.